bsip-0022.md

```diff
@@ -9,113 +9,439 @@
+|Account B|500 BTS|40%|200 BTS|200 BTS|
+|Account C|20 BTS|N/A|N/A|20 BTS|
+|**Cumulative**||||**870 BTS**|
+
+If the proxy, Account C, later _withdraws support_ for Worker Proposal 1 then the cumulative voting power for that worker proposal will be withdrawn.
+
+|Account|[Voting Stake](#account-voting-stake)|Proxy Voting Power|Proxied Votes|Voting Power for "Worker Proposal 1" after Proxy Support is Withdrawn|
+|-|-|-|-|-|
+|Account A|1000 BTS|65%|650 BTS|0 BTS|
+|Account B|500 BTS|40%|200 BTS|0 BTS|
+|Account C|20 BTS|N/A|N/A|0 BTS|
+|**Cumulative**||||**0 BTS**|
+
+This example demonstrates that even though direct votes for a worker proposal do not decay with time they can still be withdrawn at any time.
+
+---
+
+
+## <div id="vote-decay-parameters" /> Adjustable Vote Decay Parameters
+
+The BitShares Committee shall be able to adjust the three voting parameters that affect vote decay calculations across the [_two_ decayable referendum categories and proxy delegation](#referendum-categories).  The parameters within each decayable referendum category and proxy delegation are: full-power duration; total decay duration; quantity of decay steps.  There will consequently be a total of 9 parameters that can be altered by the Committee:
+
+1. full-power duration for witness slate
+2. total decay duration for witness slate
+3. quantity of decay steps for witness slate
+4. full-power duration for committee slate
+5. committee total decay duration for committee slate
+6. committee quantity of decay steps for committee slate
+7. full-power duration for proxy delegation
+8. total decay duration for proxy delegation
+9. quantity of decay steps for proxy delegation
+
+### Changing the Decay Duration of a Referendum Category
+
+![Changing decay parameters can affect an account's voting power](bsip-0022/changing-decay-parameters.png)
+
+A reduction of either full-power duration or the decay duration could have the effect of reducing the voting power of an account by effectively aging an account's vote to "expired" status from either "full-power" or "decaying" status.  A qualitative example of this can be seen in the figure if the decay parameters are changed from Settings A to Settings B.
+
+Similarly, an increase of either the full-power duration or the decay duration can increase the voting power of an account by effectively rejuvenating an account's vote from "expired" status to either "decaying" or "full-power" status, or from "decaying" to "full-power" status.  A qualitative example of this can be seen in the figure if the decay parameters are changed from Settings B to Settings A.
+
+
+### Changing the Decay Steps of a Referendum Category or Proxy Delegation
+
+Changing only the quantity of decay steps can potentially change the voting power of accounts _whose voting age fall within the decay period_.  The change in voting power for these accounts is a function of both the change in the quantity of decay steps, and the account's voting age.
+
```

Comment:
Note: the voting power of an account or proxy is recalculated during vote tallying using the chain parameter at that point in time. The chain is ignorant of the "change" in the sense that the outcome of vote tallying depends only on the *current* settings of the chain parameters, not on the settings that were valid at the previous vote tallying.